babel/index.js

```diff
@@ -333,7 +333,7 @@ BabelClient.prototype.createAnnotation = function createAnnotation(token, data,
         }
         for (var prop in target) {
             if(target.hasOwnProperty(prop)){
-                if (!(prop==="uri" || prop==="fragment" || prop==="asReferencedBy" )) {
+                if (!(prop==="uri" || prop==="fragment" || prop==="asReferencedBy" || prop === "type" )) {
```

Comment:
```suggestion
                const validProps = ['uri', 'fragment', 'asReferencedBy', 'type'];
                if (validProps.indexOf(prop) === -1) {
```

Reply:
Might also be worth adding some unit tests around this?

Reply:
https://github.com/talis/talis-node/blob/master/babel/test/unit/babel_client_test.js#L913
Looks like there are already some in this area.